src/components/Bookmark/Item/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import styles from './item.module.scss';
+import useLinkClickFix from './hooks/useLinkClickFix';
+import Favicon from '../../Favicon';
+import FaviconAuto from '../../Favicon/Auto';
+import FaviconIcon from '../../Favicon/Icon';
+import FaviconText from '../../Favicon/Text';
+import FaviconImage from '../../Favicon/Image';
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Why do we need this line?

Reply:
I disable `no-unused-vars` rule here because I exclude `id`, `row` and `column` props. Without this line, I am getting an eslint rule violation error. But I can't just remove these properties because I pass `...props` into a div and I don't want to pass unnecessary properties there.

![no-used-vars error](https://github.com/kse-web-development-practice/shcherbakov-fluent-start-page/assets/29505891/59349f14-585c-449d-af8e-f6561d3aac60)